bom/pom.xml

```diff
@@ -264,6 +279,16 @@
                 <artifactId>cxf-rt-rs-service-description</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
```

Comment:
Apologies, I missed this one, the `cxf-rt-rs-service-description-swagger` is gone in `main` (the 1.x does not support Jakarta), could you please remove it? (I will bring it back on backports). 

[1] https://github.com/apache/cxf/pull/1000

Reply:
No worries, removed!